content/docs/learning-guide/tutorials/rendering/get-started-materialtypes-and-shaders.md

```diff
@@ -343,56 +359,60 @@ This results in the following example:
 ```
 
 To recap, the above example accomplishes a couple of things:
+
 - Defines the material properties.
 - Creates a connection to the variables used by the shader source code (`.azsl`).
-- Sets up the metadata that configures how the material properties appear in the **Inspector** in **Material Editor**. 
+- Sets up the metadata that configures how the material properties appear in the **Inspector** in **Material Editor**.
 
 #### MyUnlitColor.materialtype
+
 Here is the complete `MyUnlitColor.materialtype` file:
+
 ```json
 {
-    "description": "Renders a model unlit with a single suface color value.",
```

Comment:
This following two suggestions to move the version number, fix a warning when this code is built.

```suggestion
    "description": "Renders a model unlit with a single surface color value.",
    "version": 3,
```